browser/error.ts

```diff
@@ -0,0 +1,8 @@
+export const throwNoFileSystem = (method: string) => (..._args: any[]): never => {
+	throw Object.assign(
+		new Error(
+			`Cannot access the file system (via "${method}") when using the browser build of Rollup. Make sure you supply a plugin with custom resolveId and load hooks to Rollup.`
+		),
+		{ code: 'NO_FS_IN_BROWSER', url: 'https://rollupjs.org/guide/en/#a-simple-example' }
+	);
+};
```

Comment:
For a minor opportunity for code re-use, we could consider:

```suggestion
import { error } from "../src/utils/error.ts"

export const throwNoFileSystem = (method: string) => (..._args: any[]): never => {
	error({
	  message:
	    `Cannot access the file system (via "${method}") when using the browser build of Rollup. Make sure you supply a plugin with custom resolveId and load hooks to Rollup.`,
		code: 'NO_FS_IN_BROWSER',
		url: 'https://rollupjs.org/guide/en/#a-simple-example'
  });
};
```

REF: https://github.com/rollup/rollup/blob/master/src/utils/error.ts#L13

Though can appreciate that might not be wanting `./broswer/` code to reference outside out it's own directory. That said, I'm fairly certain the browser bundle makes use of this code path via the shared core anyway.

Reply:
Good idea! No, this is totally fine to do.